src/main/java/com/unstoppabledomains/resolution/contracts/HTTPUtil.java

```diff
@@ -2,16 +2,6 @@
 
 import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-import com.unstoppabledomains.config.client.Client;
-
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.OutputStream;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
 
 public class HTTPUtil {
```

Comment:
@JohnnyJumper Can we remove this class since we're using a provider now?

Reply:
I tried to remove it, but we are using the only method that is left in this class in ZNS.java and in BaseContract.java
I think it is better if we left it as it is. In java, I can't really export a single function, since everything should be an object.